MathGame.JsPeanut/MathGame.JsPeanut/Program.cs

```diff
@@ -0,0 +1,320 @@
+                    UpdateStats(substraction);
+                }
+            }
+
+            void MultiplicationGame()
+            {
+                Operation multiplication = new Operation();
+                Console.WriteLine("You have 15 seconds to solve this operation:");
+                multiplication.Result = Multiply(a, b);
+                multiplication.Calc = ($"{a} * {b}");
+                Console.WriteLine(multiplication.Calc);
+
+                if (Console.ReadLine() == $"{multiplication.Result}")
+                {
+                    Console.WriteLine("Nice.");
+                    multiplication.Score++;
+
+                    do
+                    {
+                        a = rnd.Next(1, 100);
+                        b = rnd.Next(1, 100);
+                        UpdateStats(multiplication);
+                    } while (a + b == multiplication.Calc[0]);
+                    MultiplicationGame();
+                }
+                else
+                {
+                    Console.WriteLine("Wrong. If you typed a command, ignore this message.");
+                    PlayWithoutIntro();
+                    multiplication.Score--;
+                    UpdateStats(multiplication);
+                }
+
+
+
+            }
+
+            void DivisionGame()
+            {
+                Operation division = new Operation();
+                Console.WriteLine("You have 15 seconds to solve this operation:");
+
+                if (a % b != 0 || a == 1 || b == 1 || a == b)
+                {
+                    do
+                    {
+                        a = rnd.Next(1, 100);
+                        b = rnd.Next(1, 100);
+                    } while (a % b != 0 || a == 1 || b == 1 || a == b);
+                }
+
+                division.Result = Divide(a, b);
+                division.Calc = ($"{a} / {b}");
+
+                Console.WriteLine(division.Calc);
+
+                if (Console.ReadLine() == $"{division.Result}")
+                {
+                    Console.WriteLine("Nice.");
+                    division.Score++;
+                    UpdateStats(division);
+                    do
+                    {
+                        a = rnd.Next(1, 100);
+                        b = rnd.Next(1, 100);
+                    } while (a + b == division.Calc[0]);
+                    DivisionGame();
+                }
+                else
+                {
+                    Console.WriteLine("Wrong. If you typed a command, ignore this message.");
+                    PlayWithoutIntro();
+                    division.Score--;
+                    UpdateStats(division);
+                }
+            }
+
+            void UpdateStats(Operation operation)
+            {
+                previousGames.Add(new Operation
+                {
+                    Score = operation.Score,
+                    Result = operation.Result,
+                    Calc = operation.Calc
+                });
```

Comment:
Why do we need to reset all the values if we can just do this instead?
```suggestion
                previousGames.Add(operation);
```